trikot-viewmodels-declarative-flow/sample/common/src/commonMain/kotlin/com/mirego/sample/viewmodels/showcase/components/picker/PickerShowcaseViewModel.kt

```diff
@@ -9,4 +9,6 @@ import com.mirego.trikot.viewmodels.declarative.content.VMDTextContent
 interface PickerShowcaseViewModel : ShowcaseViewModel {
     val textPickerTitle: VMDTextViewModel
     val textPicker: VMDPickerViewModel<VMDContentPickerItemViewModelImpl<VMDTextContent>>
+    val textPickerTitle2: VMDTextViewModel
```

Comment:
We cannot re-use the same viewmodel here?

Reply:
It's just weird in the sample otherwise since both picker changes each-other's value, but we could.